content/docs/command-reference/plots/index.md

```diff
@@ -25,11 +25,20 @@ positional arguments:
 
 ## Description
 
-`dvc plots` subcommands help you visualize and compare data or images produced
-by machine learning projects, letting you fully customize graph settings and
-style.
+You can visualize and compare JSON, YAML 1.2, CSV, TSV data files or JPEG, GIF,
+PNG images found in your project. Typically these are artifacts of an [ML
+pipeline] or performance logs produced by [DVCLive].
 
-📖 See [Visualizing Plots](/doc/user-guide/visualizing-plots) for more info.
+`dvc plots` subcommands help you customize and generate these plots.
+
+📖 See [Visualizing Plots] as well as the top-level plots definition
+[specification] for more details.
```

Comment:
For the ref, I gave a bit more context so far but maybe we should restore some basic explanation of `plots` so there's no NEED to refer to the full guide or file spec.

Reply:
Let's go with this for now, but I'm fine if you want to add back more.